component/prometheus/relabel/relabel.go

```diff
@@ -46,15 +46,15 @@ type Arguments struct {
 	MetricRelabelConfigs []*flow_relabel.Config `river:"rule,block,optional"`
 
 	// Cache size to use for LRU cache.
-	//CacheSize int `river:"cache_size,attr,optional"`
+	CacheSize int `river:"max_cache_size,attr,optional"`
 }
 
 // SetToDefault implements river.Defaulter.
-/*func (arg *Arguments) SetToDefault() {
+func (arg *Arguments) SetToDefault() {
 	*arg = Arguments{
-		CacheSize: 500_000,
+		CacheSize: 100_000,
```

Comment:
Just checking - the 500k had no effect previously?

Reply:
It was commented out, the real default was 100k.